app/forms/idv/api_document_verification_form.rb

```diff
@@ -93,7 +93,7 @@ def validate_image_urls
       errors.add(:front_image_url, invalid_link) unless valid_url?(:front_image_url)
       errors.add(:back_image_url, invalid_link) unless valid_url?(:back_image_url)
       return if valid_url?(:selfie_image_url)
-      errors.add(:selfie_image_url, invalid_link) unless liveness_checking_enabled?
+      errors.add(:selfie_image_url, invalid_link) if liveness_checking_enabled?
```

Comment:
I'm so tempted to update our Rubocop to ban `unless` sometimes lol

Reply:
It definitely feels like a mind-bender at times 😅